partner-management-default.properties

```diff
@@ -258,55 +258,54 @@ mosip.role.pms.patchdevicedetail=PARTNER_ADMIN
 mosip.role.pms.postdevicedetailsearch=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
 mosip.role.pms.postdevicedetaildevicetypesearch=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
 mosip.role.pms.postdevicedetailfiltervalues=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
-mosip.role.pms.postdevicedetaildevicetypefiltervalues=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
-mosip.role.pms.postdevicedetaildevicesubtypefiltervalues=PARTNER_ADMIN,FTM_PROVIDER,DEVICE_PROVIDER
-mosip.role.pms.postdeviceprovidermanagementvalidate=ZONAL_ADMIN,ID_AUTHENTICATION,REGISTRATION_PROCESSOR,RESIDENT
+mosip.role.pms.postdevicedetaildevicesubtypefiltervalues=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
 mosip.role.pms.postftpchipdetail=PARTNER_ADMIN,FTM_PROVIDER
 mosip.role.pms.putftpchipdetail=PARTNER_ADMIN,FTM_PROVIDER
 mosip.role.pms.patchftpchipdetail=PARTNER_ADMIN
 mosip.role.pms.postftpchipdetailuploadcertificate=PARTNER_ADMIN,FTM_PROVIDER
 mosip.role.pms.getftpchipdetailgetpartnercertificate=PARTNER_ADMIN,FTM_PROVIDER
 mosip.role.pms.postftpchipdetailsearch=PARTNER_ADMIN,FTM_PROVIDER
-mosip.role.pms.postregistereddevices=PARTNER_ADMIN,FTM_PROVIDER,ZONAL_ADMIN
-mosip.role.pms.postregistereddevicesderegister=PARTNER_ADMIN,FTM_PROVIDER,ZONAL_ADMIN
-mosip.role.pms.postregistereddevicessearch=PARTNER,FTM_PROVIDER,PARTNER,AUTH_PARTNER,DEVICE_PROVIDER,CREDENTIAL_PARTNER,CREDENTIAL_ISSUANCE,CREATE_SHARE,ID_AUTHENTICATION
-mosip.role.pms.postsecurebiometricinterface=PARTNER_ADMIN,FTM_PROVIDER,DEVICE_PROVIDER
-mosip.role.pms.putsecurebiometricinterface=PARTNER_ADMIN,FTM_PROVIDER,DEVICE_PROVIDER
+mosip.role.pms.postsecurebiometricinterface=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
+mosip.role.pms.putsecurebiometricinterface=PARTNER_ADMIN,DEVICE_PROVIDER,FTM_PROVIDER
 mosip.role.pms.patchsecurebiometricinterface=PARTNER_ADMIN
-mosip.role.pms.postsecurebiometricinterfacesearch=PARTNER_ADMIN,FTM_PROVIDER,DEVICE_PROVIDER
```

Comment:
Why is the below properties removed @balaji-alluru ?

Reply:
Those properties were just added in here and not implemented or used in the code and I didn't observe this and created the new properties and added the same in here.